assets/sass/widgets/_widget-area.scss

```diff
@@ -41,6 +41,6 @@
 	}
 }
 
-.googlesitekit-widget-area--hidden {
+.googlesitekit-widget-grid-hidden {
```

Comment:
If we do use a class for this, why not use a generic display utility instead? We shouldn't really need to add a new class for hiding in each context where it's needed.